src/strobe.rs

```diff
@@ -1,7 +1,7 @@
 //! Minimal implementation of (parts of) Strobe.
 
 use keccak;
-use std::ops::{Deref, DerefMut};
+use ::core::ops::{Deref, DerefMut};
```

Comment:
rustfmt seems to want this line above `use keccak` and without the leading `::`

Reply:
Done @isislovecruft.  I did this locally while on vacation, but failed to push it :O